src/main/java/io/seqera/tower/cli/commands/LaunchCmd.java

```diff
@@ -99,6 +102,7 @@ private WorkflowLaunchRequest updateLaunchRequest(WorkflowLaunchRequest base) th
                 .id(base.getId())
                 .pipeline(base.getPipeline())
                 .computeEnvId(base.getComputeEnvId())
+                .runName(coalesce(name, base.getRunName()))
```

Comment:
What if it's empty or null?

Reply:
Coalesce should solve it, right @jordeu ?

Reply:
Yes, `coalesce` returns the first not `null` value.